### PR TITLE
release notes & changelog practices suggestions

### PR DESCRIPTION
**Pull request by petterill**
Suggestions about adhering to Keep a Changelog https://keepachangelog.com/en/1.0.0/ when doing releases. Some projects already utilize these practices at Kuva and Kanslia.
- https://github.com/City-of-Helsinki/kukkuu/releases
- https://github.com/City-of-Helsinki/kukkuu-ui/releases
- https://github.com/City-of-Helsinki/respa/releases
- https://github.com/City-of-Helsinki/varaamo/releases

### CODE REVIEW

**Comment by tituomin** [comment]
This needs discussion. It's better to pick one way of doing this instead of saying "changelog or github release notes". The two are incompatible:  

> GitHub Releases create a non-portable changelog that can only be displayed to users within the context of GitHub. It's possible to make them look very much like the Keep a Changelog format, but it tends to be a bit more involved.
>
> The current version of GitHub releases is also arguably not very discoverable by end-users, unlike the typical uppercase files (README, CONTRIBUTING, etc.). Another minor issue is that the interface doesn't currently offer links to commit logs between each release.

**Review by tituomin — CHANGES_REQUESTED** [review]
Needs one recommended way. We need to talk about it more.

**Comment by petterill** [comment]
Yes I agree, would be better to suggest only one way of doing things. @pkalliok comments?

**Comment by Jokinen** [comment]
As further context, of the example projects presented here, all but respa use both a changelog and a release. The release notes are just the content of the changelog for that version.

File: version-control.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * The branching model in use for KuVa projects is Gitflow. Refer to section [Git Workflow](#git-workflow) below for a more detailed explanation.
 * Releases are versioned using [semantic versioning](https://semver.org).
   * Note that "API changes" means e.g. breaking URL's that used to work!
+* Document changes in a changelog or in Github release notes. [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) offers a good model for informing users about changes in a release.
```
Comment (pkalliok) [inline version-control.md]:
```suggestion
* Document changes in a changelog that is called `CHANGELOG.md` and resides at the root of your repository. [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) offers a good model for informing users about changes in a release.
```

**Review by pkalliok — APPROVED** [review]
It's almost irrelevant what it is, as long as we stick to one.  I don't like the keepachangelog format and prefer debian changelog (https://manpages.debian.org/testing/dpkg-dev/deb-changelog.5.en.html), but truth be told, keepachangelog is as good as any.

**Comment by petterill** [comment]
Since the file was moved, I'll create another PR for this small change. Solving git conflicts is confusing...

**Pull request closed without merging**